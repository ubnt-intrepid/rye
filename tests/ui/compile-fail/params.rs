fn main() {}

#[rye::test]
#[rye(crate = 10)]
fn invalid_path_type() {}

#[rye::test]
#[rye(crate = "rye")]
#[rye(crate = "rye")]
fn duplicated() {}

#[rye::test]
#[rye(the_quick_fox = "lazy")]
fn unknown_param() {}
