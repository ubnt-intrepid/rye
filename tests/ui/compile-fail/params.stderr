error: required a string literal
 --> $DIR/params.rs:4:15
  |
4 | #[rye(crate = 10)]
  |               ^^

error: duplicated parameter
 --> $DIR/params.rs:9:7
  |
9 | #[rye(crate = "rye")]
  |       ^^^^^^^^^^^^^

error: unknown parameter name
  --> $DIR/params.rs:13:7
   |
13 | #[rye(the_quick_fox = "lazy")]
   |       ^^^^^^^^^^^^^
